### PR TITLE
Updated French Translation

### PR DESCRIPTION
**Pull request by Lulucmy**
### Summary:

[comment]: 
I tried the French version of the app this morning, and saw a few problems with the translation.

### Review notes (optional):
Some strings were missing I added them, I also added some words.

### Steps to test:
- Just open the app in French !

status: ready

### CODE REVIEW

**Comment by jeluard** [comment]
Please rebase on top of develop, we just merged updates to the translation (#3395)

File: src/status_im/translations/fr.cljs [hunk 1]
```diff
@@ -257,8 +274,8 @@
    :got-it                                "Compris"
    :block                                 "Bloc"
    :hash                                  "Hash"
-   :gas-limit                             "limite Gas"
-   :gas-price                             "prix Gas"
+   :gas-limit                             "limite du Gas"
```
Comment (yenda) [inline src/status_im/translations/fr.cljs:277]: limite de Gas

**Comment by Lulucmy** [comment]
Hey @jeluard ,

I will check the new file tomorrow, but what am I supposed to do with this pull request ?

**Comment by jeluard** [comment]
French translation file you modified has been changed. What you need to do is rebase your pull request to grab latest changes and resolve conflicts.

Alternatively you can re-submit your changes, as some of them have been done in between.

**Comment by jeluard** [comment]
@Lulucmy Do you need any help?

**Comment by jeluard** [comment]
Closing due to inactivity. Please reopen if it is still relevant.

**Pull request closed without merging**